testapps/testapp/src/main/res/raw/msal_config_webview.json

```diff
@@ -1,7 +1,7 @@
 {
-  "client_id" : "9851987a-55e5-46e2-8d70-75f8dc060f21",
+  "client_id" : "2523b71d-3ac3-4aea-833d-7096286ec643",
```

Comment:
Intentional?

Reply:
Could change back after Gladwin updates the redirectUri